app/collins/models/shared/IpAddressable.scala

```diff
@@ -127,11 +127,11 @@ trait IpAddressStorage[T <: IpAddressable] extends Schema with AnormAdapter[T] w
   * For a range 0L..20L, used addresses List(17,18,19,20), the result will be None (allocate from beginning)
   */
   protected def getCurrentLowestLocalMaxAddress(calc: IpAddressCalc)(implicit scope: Option[String]): Option[Long] = inTransaction {
-    val minAddress = calc.minAddressAsLong
+    val startAddress = calc.startAddressAsLong
```

Comment:
I'd prefer keeping it extremely simple and don't assume that there is a gateway nor that it would be at minAddress +1.  i.e. just document what will happen and depend on configuration to do the right thing.  That way you don't exclude interesting edge cases like I said above.